erts/doc/src/erlang.xml

```diff
@@ -6285,6 +6285,22 @@ receive_replies(ReqId, N, Acc) ->
               or <seeerl marker="#system_flag_max_heap_size">
               <c>erlang:system_flag(max_heap_size, MaxHeapSize)</c></seeerl>.</p>
           </item>
+          <tag><c>include_shared_bins</c></tag>
+          <item>
+            <p>
+	      When set to <c>true</c>, off-heap binaries are included in the
+	      total sum compared against the <c>size</c> limit. Off-heap binaries
+	      are typically larger binaries that may be shared between
+	      processes. The size of a shared binary is included by all
+	      processes that are referring it. Also, the entire size of a large
+	      binary may be included even if only a smaller part of it is
+	      referred by the process.
+            </p>
+	    <p>
+	      Default is <c>false</c>, in which case off-heap binaries are not
+	      included in the total sum.
+            </p>
```

Comment:
```suggestion
	    <p>If <c>include_shared_binaries</c> is not defined in the map, the system
              default is used. The default system default is <c>false</c>.
              It can be changed by either the option
              <seecom marker="erl#+hmaxib">+hmaxib</seecom> in <c>erl(1)</c>,
              or <seeerl marker="#system_flag_max_heap_size">
              <c>erlang:system_flag(max_heap_size, MaxHeapSize)</c></seeerl>.
            </p>
```